avm/res/service-fabric/cluster/tests/e2e/max/main.test.bicep

```diff
@@ -95,16 +90,7 @@ module testDeployment '../../../main.bicep' = [
           isAdmin: false
         }
       ]
-      clientCertificateThumbprints: [
```

Comment:
Same as above for this removed parameter

Reply:
clientCertificateThumbprints cannot be used when clientCertificateCommonNames is specified.

Reply:
Hey @lsnoddy, can we please add that and similar info to the parameter description metadata? It would help module consumers to know about conflicting inputs

Reply:
@eriqua , I updated the descriptions for clientCertificateThumbprints and clientCertificateCommonNames to include this information. Please check.